infrastructure/modules/cloudflare_worker_static_pages/main.tf

```diff
@@ -0,0 +1,51 @@
+terraform {
+  required_providers {
+    cloudflare = {
+      source  = "cloudflare/cloudflare"
+      version = "~> 2.0"
+    }
+  }
+}
+
+resource "cloudflare_workers_kv_namespace" "static_content" {
+  title = "${var.hostname}/static-content"
+}
+
+resource "null_resource" "kv_static_content" {
```

Comment:
issue to create a custom provider to create KV content without polluting terraform state

Reply:
#37 raised for this